### PR TITLE
updating usage for runc, and all runc commands that now use <container id> as the first argument

### PR DESCRIPTION
**Pull request by mikebrow**
Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

**Comment by jfrazelle** [comment]
ugh sorry these are parsing horribly I can fix tonight or tomorrow

On Wed, Feb 10, 2016 at 2:13 PM, Gordon <notifications@github.com> wrote:

> Job: runc-PRs FAILED
> <https://jenkins.dockerproject.org/job/runc-PRs/1162/console>:
>
> ---d image 'busybox:latest' locallylatest: Pulling from library/busybox583635769552: Pulling fs layerb175bcb79023: Pulling fs layerb175bcb79023: Verifying Checksumb175bcb79023: Download complete583635769552: Verifying Checksum583635769552: Download complete583635769552: Pull completeb175bcb79023: Pull completeDigest: sha256:c1bc9b4bffe665bf014a305cc6cf3bca0e6effeb69d681d7a208ce741dad58e0Status: Downloaded newer image for busybox:latestBuild step 'Execute shell' marked build as failure[PostBuildScript] - Execution post build scripts.[runc-PRs] $ /bin/sh -xe /tmp/hudson8855446973557951806.sh+ set +x
>
> ---
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runc/pull/546#issuecomment-182535101>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by crosbymichael** [comment]
the formatting for all these strings are pretty bad because of the newlines....

**Comment by mikebrow** [comment]
@crosbymichael re formating with newlines issue.  Agreed.  

I was following the pattern we had from main.go for runc... If I take the newlines out the result is the first line is spaced over like a paragraph start and the line wrap then starts at the far left. Which does not look right.

Options: 
1) override/change the templates in github.com/codegangsta/cli/help.go to be left justified vs three spaces over... 
2) Leave the difficult to format strings as is in this PR so far...
3) Remove most of the new lines from the usage paragraphs and live with the three spaces on the start of each paragraph line style. Where we would only have difficult formatting on new paragraphs.

WDYT?

**Comment by crosbymichael** [comment]
Well right now the commands are inconsistent with the main.go because they all have spaces before each line.  We should move them all the way to the left and not have any tabs/spaces before the sentences

**Comment by mikebrow** [comment]
@crosbymichael  Done....  Cheers.

**Comment by marcosnils** [comment]
=1

**Comment by crosbymichael** [comment]
LGTM

File: main.go [hunk 1]
```diff
@@ -23,14 +23,18 @@ container runtime environment for applications. It can be used with your
 existing process monitoring tools and the container will be spawned as a
 direct child of the process supervisor.
 
-After creating config files for your root filesystem with runc, you can execute 
-a container in your shell by running:
+Containers are configured using using bundles. A bundle for a container is a
```
Comment (mrunalp) [inline main.go]: s/using using/using/

File: exec.go [hunk 2]
```diff
@@ -19,6 +19,15 @@ import (
 var execCommand = cli.Command{
 	Name:  "exec",
 	Usage: "execute new process inside the container",
+	ArgsUsage: `<container-id> <container command>
+
+Where "<container-id>" is the name for the instance of the container and
+"<container command>" is the command to be executed in the container.
+
+For example, if the container is a linux distro the following will output a list
```
Comment (mrunalp) [inline exec.go]: I would reword this as the container doesn't necessarily have to be a distro. Even a busybox rootfs has the ps command.

File: main.go (hunk 1 above)
Comment (mikebrow) [inline main.go]: Fixed, thx.

File: exec.go (hunk 2 above)
Comment (mikebrow) [inline exec.go]: Reworded.

**Comment by mikebrow** [comment]
@mrunalp your comments have been addressed.  Cheers!

**Comment by mrunalp** [comment]
@mikebrow Thanks!

**Comment by mrunalp** [comment]
LGTM
